src/modules/fancyzones/dll/dllmain.cpp

```diff
@@ -332,23 +292,15 @@ void FancyZonesModule::HandleWinHookEvent(WinHookEvent* data) noexcept
 
 void FancyZonesModule::MoveSizeStart(HWND window, POINT const& ptScreen) noexcept
 {
-    if (IsInterestingWindow(window))
+    if (auto monitor = MonitorFromPoint(ptScreen, MONITOR_DEFAULTTONULL))

```

Comment:
Instead of doing the `IsInterestingWindow` check here, we do it in the `lib/FancyZones.cpp`